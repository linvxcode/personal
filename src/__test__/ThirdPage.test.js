

import React from "react";
import { render, screen} from "@testing-library/react";
import ThirdPage from "../Third Page/ThirdPage";

describe('ThirdPage', () => {
    it('Render', () => {
        const {PageRender} = render(<ThirdPage/>)
        expect(PageRender).toMatchSnapshot();
        const element = screen.getByText('Techno');
        expect(element).toBeInTheDocument()
    })
})