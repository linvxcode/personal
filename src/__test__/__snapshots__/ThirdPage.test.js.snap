// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThirdPage Render 1`] = `undefined`;
